cmd/rook-operator/restapi.go

```diff
@@ -65,11 +65,11 @@ func startAPI(cmd *cobra.Command, args []string) error {
 	context := clusterd.NewDaemonContext(cfg.dataDir, cfg.cephConfigOverride, cfg.logLevel)
 	factory := cephd.New()
 	apiCfg := &api.Config{
-		ConnFactory:    mon.NewConnectionFactoryWithClusterInfo(&cfg.clusterInfo),
-		CephFactory:    factory,
-		Port:           apiPort,
-		ClusterHandler: apik8s.New(clientset, context, &cfg.clusterInfo, factory, cfg.namespace, cfg.containerVersion),
-		InitRetryMs:    5000,
+		ConnFactory:            mon.NewConnectionFactoryWithClusterInfo(&cfg.clusterInfo),
+		CephFactory:            factory,
+		Port:                   apiPort,
+		ClusterHandler:         apik8s.New(clientset, context, &cfg.clusterInfo, factory, cfg.namespace, cfg.containerVersion),
+		RegisterMetricsRetryMs: 5000,
```

Comment:
that's actually a good point.  the intent was not to make it user configurable but to instead enable production code to have a different value than unit test code.  now that i've settled on how the `RegisterMetrics` function is invoked, it doesn't need to be part of the config.  It could just be an argument to the function.  I'll make that change then squash and merge.  Thanks!